### PR TITLE
rdgo: Publish repo to S3

### PR DESCRIPTION
**Pull request by jlebon**
These are based on the git master of various projects and thus should
just be made public for easier consumption.

### CODE REVIEW

File: Jenkinsfile.rdgo [hunk 1]
```diff
@@ -13,6 +13,7 @@ node(env.NODE) {
                 cp RPM-GPG-* /etc/pki/rpm-gpg/
                 dnf copr -y enable walters/buildtools-fedora
                 dnf install -y rpmdistro-gitoverlay
+                pip3 install awscli==1.15.20
```
Comment (cgwalters) [inline Jenkinsfile.rdgo]: FWIW this also exists as `awscli` as an RPM.

File: Jenkinsfile.rdgo [hunk 2]
```diff
@@ -63,6 +64,14 @@ node(env.NODE) {
                         ${rdgo}/ ${env.ARTIFACT_SERVER}:${rdgo}
                 """
             }
+            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding',
+                              credentialsId: env['AWS_CREDENTIALS'],
+                              accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                              secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                sh """
+                    aws s3 sync ${rdgo}/build/ s3://aos-ci/rhcos/rdgo
```
Comment (cgwalters) [inline Jenkinsfile.rdgo]: I notice we aren't using `--delete`...that's probably OK short term.  But I feel like we should set up something like Pulp or whatever to push to - that also applies to origin RPMs.

**Comment by cgwalters** [comment]
Basically seems OK, but leaving unmerged in case you want to change anything.  If not then I'll lgtm.

File: Jenkinsfile.rdgo (hunk 1 above)
Comment (jlebon) [reply Jenkinsfile.rdgo]: Ahh, nice. I had checked `python-awscli` and `python3-awscli`, but not `awscli` directly.

File: Jenkinsfile.rdgo (hunk 2 above)
Comment (jlebon) [reply Jenkinsfile.rdgo]: Good point, the CLI supports `--delete`, so added in :arrow_down:

**Comment by jlebon** [comment]
I also dropped the

```
accessKeyVariable: 'AWS_ACCESS_KEY_ID',
secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
```

bits since those are the default values already.

**Pull request merged**